packages/npm-package-json-lint-config/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Bug Fix
+
+-   Packages: Replace `is-plain-obj` with `is-plain-object` ([#43511](https://github.com/WordPress/gutenberg/pull/43511)).
```

Comment:
Looks like this one is replacing it with native checks rather than `is-plain-object` now.

Reply:
Good spot, thanks! Addressed as part of 3b02f0048724f064dcd1df8f6968d7e38c1565fb.